### PR TITLE
fix: dropdown flashing in collapsible section view

### PR DESCRIPTION
**Pull request by wegry**
https://www.loom.com/share/f2ada147cd794f4f9d4ce25f74c7129a

# Before
Screenshot 2024-01-25 at 16 08 28

# After

Screenshot 2024-01-25 at 16 07 39

### CODE REVIEW

File: src/platform-implementation-js/style/gmail.css [hunk 1]
```diff
@@ -2232,8 +2222,8 @@ shows up on our elements preventing them from collapsing.
   margin-right: 0px;
 }
 
-.inboxsdk__resultsSection_collapsed .Cr {
-  display: none;
+.inboxsdk__resultsSection_collapsed .inboxsdk__resultsSection_header {
```
Comment (wegry) [inline src/platform-implementation-js/style/gmail.css:2225]:
```suggestion
/* 2024-01-25 Prevent the addition of extra padding on the right side of the header from `.Wg` */
.inboxsdk__resultsSection_collapsed .inboxsdk__resultsSection_header {
```

File: examples/search-results/content.js [hunk 2]
```diff
@@ -1,9 +1,100 @@
...
+
 InboxSDK.load(1.0, 'search-example').then(function (inboxSDK) {
+  inboxSDK.Router.handleListRoute(
+    inboxSDK.Router.NativeRouteIDs.INBOX,
+    (listRouteView) => {
+      listRouteView.addCollapsibleSection({
+        title: 'Inbox Monkeys',
+        titleLinkText: 'View All',
+        hasDropdown: true,
+        onDropdownClick: () => {
```
Comment (wegry) [inline examples/search-results/content.js:11]: From @ruthhalteman 

```suggestion
        /** A dropdown would go here, but isn't hooked up in this example. */
        onDropdownClick: () => {
```

File: src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts [hunk 3]
```diff
@@ -625,7 +638,7 @@ class GmailCollapsibleSectionView {
   #addToCollapsedContainer() {
     const element = this.#element;
     if (!element) return;
-    if (this.#headerElement) this.#headerElement.classList.remove('Wg');
+    // if (this.#headerElement) this.#headerElement.classList.remove('Wg');
```
Comment (ruthhalteman) [inline src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts:641]: should this just be deleted? if not a comment to say why?

**Review by ruthhalteman — APPROVED** [review]

File: src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts [hunk 4]
```diff
@@ -15,6 +15,16 @@ import type {
   SectionDescriptor,
 } from '../../../../inboxsdk';
 
+const enum GmailClass {
```
Comment (ruthhalteman) [inline src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts:18]: this stuff makes it so much clearer when reading the code 🙏

File: src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts (hunk 3 above)
Comment (wegry) [reply src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts]: Removed.

**Pull request merged**